.github/workflows/backport.yml

```diff
@@ -0,0 +1,26 @@
+name: Backport
+on:
+  pull_request_target:
```

Comment:
Ooo, yeah gotta be careful with this one. I'll need to re-learn a few things.

The docs on tibdex/backport are pretty light: did you follow something to accomplish this? Do you know if backports open PRs, or do they just automatically cherry-pick?

Reply:
> did you follow something to accomplish this?

Not really, just searched for the action to do this with most stars :sweat_smile: 

> Do you know if backports open PRs, or do they just automatically cherry-pick?

As I understand it, it creates new PRs; check https://github.com/tibdex/backport/blob/main/src/backport.ts#L268

Reply:
By the way, there's much more info in the [GitHub Marketplace page of the App](https://github.com/marketplace/actions/backporting-github-action#how-it-works).

Reply:
Did you have time to check this?

Reply:
Gently ping @kyrofa

Reply:
@pachulo don't wait for me, feel free to land and iterate as necessary.